.azure-pipelines/docker-sonic-mgmt.yml

```diff
@@ -55,14 +61,42 @@ stages:
       env:
         REGISTRY_SERVER: ${{ parameters.registry_url }}
       displayName: Build docker-sonic-mgmt.gz
+
+    - publish:  $(Build.ArtifactStagingDirectory)
+      artifact: 'docker-sonic-mgmt'
+      displayName: "Archive docker image sonic-mgmt"
+
+  - job: validate_docker_image_and_upload
+    pool: sonicbld-1es
+    timeoutInMinutes: 360
+    dependsOn: Build
+    steps:
+    - checkout: sonic-mgmt
+      clean: true
+      displayName: 'Checkout sonic-mgmt'
+
+    - download: current
+      artifact: 'docker-sonic-mgmt'
+      displayName: "Download docker image sonic-mgmt"
+
+    - script: |
+        set -ex
+
+        docker load -i $(Pipeline.Workspace)/docker-sonic-mgmt/target/docker-sonic-mgmt.gz
+
+        cd ansible
+        sudo ./setup-management-network.sh -d
+        cd ../
+        docker rm -f sonic-mgmt
+
+        ./setup-container.sh -n sonic-mgmt -d /data -i docker-sonic-mgmt -v
```

Comment:
How about run something inside the docker to verify that the sonic-mgmt container is actually running?